srtcore/group.cpp

```diff
@@ -448,10 +448,12 @@ template <class ValueType>
 static void importOption(vector<CUDTGroup::ConfigItem>& storage, SRT_SOCKOPT optname, const ValueType& field)
 {
     ValueType default_opt      = ValueType();
-    int       default_opt_size = sizeof(ValueType);
+    const int default_opt_size = sizeof(ValueType);
```

Comment:
Here you can even make it a static const.

Reply:
Changed to static 👍